packages/elements/src/tsconfig.json

```diff
@@ -6,13 +6,5 @@
     "module": "es2020",
     "moduleResolution": "node",
     "lib": ["es2020", "dom", "dom.iterable"]
-  },
-  "references": [
-    {
-      "path": "../../report-schema/src"
-    },
-    {
-      "path": "../../metrics/src"
-    }
-  ]
```

Comment:
Why aren't we using native project references anymore? Sounds like a step backwards.

Reply:
Agreed, so I reverted this